numpy/core/src/multiarray/common.h

```diff
@@ -2,6 +2,7 @@
 #define _NPY_PRIVATE_COMMON_H_
 #include <numpy/npy_common.h>
 #include <numpy/npy_cpu.h>
+#include <numpy/npy_3kcompat.h>
```

Comment:
This include seems to open a can of worms, but I have no idea why. All I wanted was `PyUString_FromString`!

Reply:
What and where to include remains a complete mystery to me. I usually start with something that has everything and then delete all lines that do not cause compilation errors...

Reply:
Our include files are a tangle that has grown over the years. Someday we should rationalize them ...